org.jousse.vincent.Pomodorolm.yml

```diff
@@ -28,7 +28,7 @@ modules:
     sources:
       - type: git
         url: https://github.com/vjousse/pomodorolm.git
-        branch: flathub
+        tag: flathub
```

Comment:
lying doesn't help

![image](https://github.com/user-attachments/assets/f276c2ba-034e-41cb-8fb3-62bf2dcbad55)

Reply:
```suggestion
```

Reply:
What are you talking about and why are you so aggressive?
![image](https://github.com/user-attachments/assets/276fa517-748b-413b-b827-add09c1420e8)

Just trying to do my best here, no need to YELL at me in your previous comment nor to say that I'm a liar when I'm not.